static/src/js/components/GranuleResults/__tests__/GranuleResultsItem.test.js

```diff
@@ -335,316 +363,417 @@ function setup(type, overrideProps) {
           }
         ],
         s3Links: []
-      },
-      ...overrideProps
+      }
     }
   }
 
-  const enzymeWrapper = shallow(<GranuleResultsItem {...props} />)
+  props = {
+    ...props,
+    ...overrideProps
+  }
+
+  render(
+    <GranuleResultsItem {...props} />,
+    { wrapper: MemoryRouter }
+  )
 
   return {
-    enzymeWrapper,
+    user,
     props
   }
 }
 
 beforeEach(() => {
   jest.clearAllMocks()
+
+  ReactDOM.createPortal = jest.fn((dropdown) => dropdown)
 })
 
 describe('GranuleResultsItem component', () => {
   test('renders itself correctly', () => {
-    const { enzymeWrapper } = setup('cmr')
+    setup('cmr')
 
-    expect(enzymeWrapper.exists()).toEqual(true)
-    expect(enzymeWrapper.type()).toBe('div')
+    expect(screen.getByTestId('granule-results-item')).toBeInTheDocument()
   })
 
-  test('renders the add button under PortalFeatureContainer', () => {
-    const { enzymeWrapper } = setup('cmr')
+  test('renders the add button under PortalFeatureContainer', async () => {
+    setup('cmr')
 
-    const button = enzymeWrapper
-      .find(PortalFeatureContainer)
-      .find('.granule-results-item__button--add')
-    const portalFeatureContainer = button.parents(PortalFeatureContainer)
-
-    expect(button.exists()).toBeTruthy()
-    expect(portalFeatureContainer.props().authentication).toBeTruthy()
+    await waitFor(() => {
+      expect(screen.getByRole('button', { name: 'Add granule' })).toBeInTheDocument()
+    })
   })
 
   describe('when passed a focused granule', () => {
     test('adds the correct classname', () => {
-      const {
-        enzymeWrapper
-      } = setup('focused-granule')
+      setup('focused-granule')
 
-      expect(enzymeWrapper.props().className).toContain('granule-results-item--active')
+      expect(screen.getByTestId('granule-results-item')).toHaveClass('granule-results-item--active')
```

Comment:
Avoid using test ids, need to find a way to find this element in a way that better simulates how a user would use the app